ppafm/HighLevel.py

```diff
@@ -54,13 +54,13 @@ def relaxedScan3D( xTips, yTips, zTips, trj=None, bF3d=False ):
     for ix,x in enumerate( xTips  ):
         sys.stdout.write('\033[K')
         sys.stdout.flush()
-        sys.stdout.write("\rrelax ix: {}".format(ix))
+        sys.stdout.write(f"\rrelax ix: {ix}")
         sys.stdout.flush()
         for iy,y in enumerate( yTips  ):
             rTips[:,0] = trj[:,0] + x
             rTips[:,1] = trj[:,1] + y
             rTips[:,2] = trj[:,2]
-            itrav = core.relaxTipStroke( rTips, rs, fs ) / float( len(zTips) )
+            core.relaxTipStroke( rTips, rs, fs ) / float( len(zTips) )
```

Comment:
Hey, really, really sorry for being slow (as I have told, this has lower priority than my immediate work, with 2 new students, that I am advising ...) - we spoke about this particular line yesterday with @ProkopHapala - from my point of view, the `/ float( len(zTips) )` is not necessary at all.